axelor-production/src/main/resources/reports/ManufOrder.rptdesign

```diff
@@ -1353,6 +1353,13 @@ INNER JOIN
                     <property name="dataType">boolean</property>
                     <property name="allowExport">true</property>
                 </structure>
+                <structure>
+                    <property name="name">manuf_note</property>
+                    <text-property name="displayName">manuf_note</text-property>
+                    <expression name="expression" type="javascript">"Note : "+dataSetRow['note']</expression>
+                    <property name="dataType">string</property>
```

Comment:
Display ahead means before above not below, also don't display title for 'Note' and it should not display null.